beacon-chain/rpc/eth/config/handlers_test.go

```diff
@@ -173,7 +173,7 @@ func TestGetSpec(t *testing.T) {
 	data, ok := resp.Data.(map[string]interface{})
 	require.Equal(t, true, ok)
 
-	assert.Equal(t, 132, len(data))
```

Comment:
what changed here? was this just broken?

Reply:
I removed `DeprecatedSafeSlotsToUpdateJustified` and `DeprecatedSafeSlotsToImportOptimistically` from `config/params/config.go`